### PR TITLE
Update readme of nginx-ingress with L4 mapping

### PR DESCRIPTION
**Pull request by zenglian**
Update readme of nginx-ingress with L4 mapping example

#### What this PR does / why we need it:
Saved a lot of searching time.

#### Which issue this PR fixes
Resovle https://github.com/helm/charts/issues/5408

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by ChiefAlexander** [comment]
Thanks for the PR! Sorry that I just was able to find it. For us to merge this in we need todo two things. The first is that your commit has to be signed which is detailed in https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work
tldr: run `git commit --amend --signoff` for previous commits and `git commit -s` when doing new commits.
Finally, even though this is just a documentation change linting and tests will fail until you bump the chart version.
/assign

**Comment by zenglian** [comment]
It's a minor doc change. Please check and add it along with your commits if appropriate.
